src/Libraries/Watch3DNodeModelsWpf/HelixWatch3DNodeViewModel.cs

```diff
@@ -116,11 +116,11 @@ protected override void PortDisconnectedHandler(PortModel obj)
         {
             if (obj.PortType == PortType.Input  && watchModel == obj.Owner)
             {
-                OnClear();
+                UpdateUpstream();
```

Comment:
what is this change for?

Reply:
Without this change, the view inside the node will not be updated when a port is disconnected.